### DISCUSSION TITLE
Mobile Authentication with Microsoft SSO account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#24063] [Enterprise] [answered]
<p>My client is using Github Enterprise but we don’t have a specific server URL for it, which is a required field when trying to login with Github Enterprise on mobile. There for I am not able to authenitcation with Github Enterprise and I am also not able to authenticate with a username/password. We are currently using Microsoft SSO on the browser. Any suggestions on how I get GitHub Mobile working for this account?</p>

**Comment by ghost** [comment]
<p>We have the exact same challenges and would love some guidance on this.<br>
Using GitHub enterprise cloud in general has very little documentation for what to do with an SSO flow.</p>

**Comment by ghost** [comment, marked as answer]
<p>I asked github support and got a reply.</p>
<p>When prompted to choose between <a href="http://Github.com" rel="noopener nofollow ugc">Github.com</a> and Github Enterprise server, select <a href="http://Github.com" rel="noopener nofollow ugc">Github.com</a>.</p>
<p>Then, in the Username and Password tabs, enter your github SSO username.<br>
When you have entered the SSO username, this changes the login button to say “Sign in with your identity provider”.</p>
<p>When you enter that button you are then prompted to authenticate via your SSO flow.<br>
I tested this and it worked for our users using Azure AD as our SSO / IDP provider with Github Enterprise Cloud.</p>

**Comment by tuves** [comment]
<p>Thank you for circling back with the solution, <a class="mention" href="/hsmx_nngithub">@HSMX_nngithub</a>! :smile:</p>
